.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,6 +24,7 @@
 
 ## Quality Assurance
 
+- [ ] Ran `yarn changeset`
```

Comment:
Some PRs (such as this one) won't need a changeset.
What if the wording was instead:
```suggestion
- [ ] Ran `yarn changeset` if adapter source code has changes
```